packages/chain-adapters/src/cosmossdk/thorchain/ThorchainChainAdapter.ts

```diff
@@ -207,7 +207,9 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<KnownChainIds.ThorchainMa
       if (!fee) throw new Error('fee is required')
 
       const account = await this.getAccount(from)
-      const amount = this.getAmount({ account, value, fee, sendMax })
+      // Never deduct value for native, non-fee assets. Max-send fees deduction only apply to fee asset i.e THOR.RUNE here
+      // THOR.TCY is a native asset, but not a fee asset, for all intents and purposes it's a token
+      const amount = coin === 'THOR.RUNE' ? this.getAmount({ account, value, fee, sendMax }) : value
```

Comment:
@kaladinlight FYI, we will need to do the same in MAYA adapter as soon as we start handling both `coin`s there or we'll have the same bug for MAYA vs CACAO🙏🏽